### DISCUSSION TITLE
Scrap furnishing so we can import it into paimon.moe or other website if there is any

### DISCUSSION THREAD

**Discussion by KamilNiemcewicz** [Andrewthe13th/Inventory_Kamera#491] [Ideas or Feature requests]
Just as in the title - since the whole scrapper already exist for many other things, would you like to expand it for furnishing? Having hundreds of materials already crafted it would be an unbelievable pain to go through all of them and mark how many is crafted and how many is needed for all furnishing sets to complete. It would also allow to plan better any designs 😄

**Comment by Dyl-Aeon** [comment]
Seconding this. I wonder what could be done to query known recipes/set lists as well. Probably go item by item in the "Create Furnishing" page for the recipes, but that'd require a different starting point than the Paimon menu.
